lib/lita/handler/chat_router.rb

```diff
@@ -103,7 +103,7 @@ def dispatch_to_route(route, robot, message)
         handler = new(robot)
         route.callback.call(handler, response)
       rescue => error
-        log_error(robot, error)
+        log_error(robot, error, message)
```

Comment:
As discussed internally, I think you are better to pass a hash of metadata, because the handle might be called in other contexts, like from an HTTP route or something.

It would also allow to add other metadata in the future without changing the interface.

```ruby
log_error(robot, error, message: message)
```